client/less/classic-modal.less

```diff
@@ -0,0 +1,45 @@
+.success {
```

Comment:
I have applied all the classes I can find on the modal (bar animation specific) on the current production site.

These classes are still needed